src/blocks/Article/InfoBox/config.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  lexicalEditor,
+  HeadingFeature,
+  FixedToolbarFeature,
+  InlineToolbarFeature,
+  OrderedListFeature,
+  UnorderedListFeature,
+} from '@payloadcms/richtext-lexical';
+import type { Block } from 'payload';
+
+export const InfoBox: Block = {
+  slug: 'infoBox',
+  interfaceName: 'infoBoxBlock',
+  fields: [
+    {
+      name: 'richText',
+      type: 'richText',
+      label: false,
+      required: true,
+      editor: lexicalEditor({
+        features: ({ rootFeatures }) => {
+          return [
+            ...rootFeatures,
+            HeadingFeature({ enabledHeadingSizes: ['h5'] }),
+            FixedToolbarFeature(),
+            InlineToolbarFeature(),
+            UnorderedListFeature(),
+            OrderedListFeature(),
+          ];
+        },
+      }),
+    },
+    {
+      name: 'backgroundColor',
+      label: 'Background color',
+      type: 'select',
+      options: [
+        {
+          value: 'greenPurple',
+          label: 'Green/Purple',
+        },
+        {
+          value: 'greenRed',
+          label: 'Green/Red',
+        },
+      ],
```

Comment:
Same question as in other PR. Is this up to editor, or are there some guidelines?